### DISCUSSION TITLE
Disable or Override Notification Prompt for MediaElement on Android

### DISCUSSION THREAD

**Discussion by vikher** [CommunityToolkit/Maui#2074] [General]
I have an app where push notifications are used, and I've encountered an issue with the MediaElement component on Android. My app redirects users to settings to enable push notifications, but if they start the app and go directly to play a video, the MediaElement prompt for notifications is misleading. This is because selecting "Don't Allow" or "Allow" affects the notifications settings, which can confuse users.

Is there a way to disable or override this prompt for the MediaElement on Android?

![image](https://github.com/user-attachments/assets/3bd85ff2-863a-41ee-8cb7-3124f2f89c00)

**Comment by ne0rrmatrix** [comment]
I updated media3 PR here: #2076 to no longer request access to notifications. There is no longer any code that calls for access or requests notifications at all in media element. It only requires request for service in manifest and no longer has any need for notification access. It still uses notifications but it lets OS do that and no user interaction is required.

**Reply by bijington** [reply]
I am not sure it needs to be part of the migration PR but I think we should make things like notifications opt-in rather than on by default.

**Reply by ne0rrmatrix** [reply]
Do u mean have media controls in notification tray an opt in only option?

**Reply by bijington** [reply]
Yes
